### PR TITLE
Fix android custom networks list

### PR DESCRIPTION
**Pull request by supermassive**
Resolves https://github.com/brave/brave-browser/issues/23205

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: browser/ui/android/strings/android_brave_strings.grd [hunk 1]
```diff
@@ -1420,12 +1420,6 @@ Are you sure you want to do this?
       <message name="IDS_RETRY" desc="Retry button text.">
         Retry
       </message>
-      <message name="IDS_PRIVACY_STATS" desc="Private stats title text.">
```
Comment (supermassive) [inline browser/ui/android/strings/android_brave_strings.grd:1423]: references to these were removed here https://github.com/brave/brave-core/pull/12743/files#diff-4326ff8a702dc0509f107bbfac92c3814a71d1e6bc8f43cc50f626adb45fb216

**Review by SergeyZhukovsky — APPROVED** [review]
++, thanks for handling it on Android

File: android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java [hunk 2]
```diff
@@ -359,6 +383,24 @@ public static CharSequence getNetworkShortText(Activity activity, String chainId
             case BraveWalletConstants.LOCALHOST_CHAIN_ID:
                 strNetwork = activity.getText(R.string.localhost);
                 break;
+            case BraveWalletConstants.POLYGON_CHAIN_ID:
+                strNetwork = activity.getText(R.string.polygon_short;
```
Comment (SergeyZhukovsky) [inline android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java:387]: missed `)`

**Comment by yrliou** [comment]
x86 test failure is unrelated  
```
1 test timed out:
[2022-06-03T14:51:25.570Z]     BraveSchemeLoadBrowserTest.CrashURLTest (../../brave/browser/brave_scheme_load_browsertest.cc:208)
```

**Pull request merged**